shotover-proxy/src/transforms/cassandra/sink_cluster/node_pool.rs

```diff
@@ -171,12 +172,14 @@ impl NodePool {
                 Murmur3Token::generate(&routing_key),
                 keyspace.replication_factor,
             )
+            .unique()
             .collect::<Vec<uuid::Uuid>>();
 
         let (dc_replicas, rack_replicas) = self
             .nodes
             .iter_mut()
             .filter(|node| replica_host_ids.contains(&node.host_id) && node.is_up)
+            .unique_by(|node| node.rack.clone())
```

Comment:
Wouldnt this result in rack_replicas always containing 0 or 1 elements?

Reply:
Yes

Reply:
In that case should we treat rack_replicas as having one value?
![image](https://user-images.githubusercontent.com/5120858/202597808-4a7a67fa-db39-47fb-a2b5-32fe43072418.png)